plugins/filter_aws/aws.c

```diff
@@ -31,22 +31,39 @@
 #include <fluent-bit/flb_io.h>
 #include <fluent-bit/flb_kv.h>
 
+
 #include <monkey/mk_core/mk_list.h>
 #include <msgpack.h>
-
 #include <stdlib.h>
 #include <errno.h>
 
 #include "aws.h"
 
 static int get_ec2_token(struct flb_filter_aws *ctx);
-static int get_metadata(struct flb_filter_aws *ctx, char *metadata_path,
-                        flb_sds_t *metadata, size_t *metadata_len);
+static int get_metadata(struct flb_filter_aws *ctx, char *metadata_path, flb_sds_t *metadata, size_t *metadata_len);
+static int get_metadata_by_key(struct flb_filter_aws *ctx, char *metadata_path,
+                        flb_sds_t *metadata, size_t *metadata_len, char *key);
```

Comment:
Align the indented code to the opening `(`

Also note: try to keep every line of code less than 80 characters in length, as noted in they style guide (I should've sent you this earlier): https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#coding-style

Reply:
Good call! Modified the code to follow the code style guidance.